userspace/libsinsp/container.cpp

```diff
@@ -88,7 +88,7 @@ bool sinsp_container_manager::remove_inactive_containers()
 		});
 
 		auto containers = m_containers.lock();
-		if (m_inspector->m_sinsp_stats_v2)
+		if (m_inspector != nullptr && m_inspector->m_sinsp_stats_v2)
```

Comment:
In line `69` we dereference the inspector :thinking: 
```cpp
	if(m_inspector->m_lastevent_ts >
		m_last_flush_time_ns + m_inspector->m_inactive_container_scan_time_ns)
		m_last_flush_time_ns + m_inspector->m_inactive_container_scan_time_ns)
```
Not sure if we want to add an extra check also above of remove these checks like before

Reply:
it is also used at line `66`